api/api.go

```diff
@@ -207,6 +207,7 @@ func New(requiredUserAgent string, requiredPassword string, cs modules.Consensus
 		router.GET("/renter", api.renterHandlerGET)
 		router.POST("/renter", RequirePassword(api.renterHandlerPOST, requiredPassword))
 		router.GET("/renter/contracts", api.renterContractsHandler)
+		router.GET("/renter/contract/*contractid", api.renterContractDetailHandler)
```

Comment:
per convention, the name of the flag would be `includehistorical`.

We will add a version 2 of the api at somepoint, but definitely it does not make sense at this point. I'm guessing it will not make sense for at least 12 - 18 more months, potentially longer